backend/app/services/knowledge/knowledge_service.py

```
@@ -1640,6 +1646,218 @@ def get_personal_knowledge_bases_grouped(
+            )
+            .all()
+        )
+        shared_kb_ids = [p[0] for p in shared_kb_ids]
+
+        personal_shared = []
+        if shared_kb_ids:
+            personal_shared = (
+                db.query(Kind)
+                .filter(
+                    Kind.kind == "KnowledgeBase",
+                    Kind.is_active == True,
```

Ruff (0.15.6)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)